bigquery/samples/tests/test_download_public_data.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import download_public_data
+
+
+def test_download_public_data(capsys, client):
+
+    download_public_data.download_public_data(client)
+    out, _ = capsys.readouterr()
+    assert "year" in out
+    assert "gender" in out
+    assert "name" in out
```

Comment:
How does this check that the BQ storage client was indeed used? Or is it solely for making sure that the sample indeed runs without errors?

(the same for the sandbox version of the sample)

**Edit:** I see that the related unit tests cover this aspect. Let's maybe just add a comment about that here to avoid any doubts from the future readers?

Reply:
I've updated the test to verify that a BigQuery Storage API session appears in the DEBUG logs.